src/Adapters/Facebook/OEmbed.php

```diff
@@ -21,6 +21,12 @@ protected function detectEndpoint(): ?UriInterface
         }
 
         $uri = $this->extractor->getUri();
+        if (strpos($uri->getPath(), 'login') !== false) {
+            parse_str($uri->getQuery(), $params);
+            if (isset($params['next']) && $params['next']) {
```

Comment:
A shorter way to make this is:
```suggestion
            if (!empty($params['next'])) {
```